test_runner/src/main/kotlin/ftl/json/OutcomeDetailsFormatter.kt

```diff
@@ -12,8 +12,10 @@ import ftl.util.StepOutcome.skipped
 import ftl.util.StepOutcome.success
 import ftl.util.StepOutcome.unset
 
-fun Outcome.getDetails(testSuiteOverviewData: TestSuiteOverviewData?) = when (summary) {
-    success, flaky -> testSuiteOverviewData?.getSuccessOutcomeDetails(successDetail?.otherNativeCrash ?: false)
+internal fun Outcome.getDetails(testSuiteOverviewData: TestSuiteOverviewData?): String = when (summary) {
```

Comment:
you could get rid of return type `:String`

Reply:
Right, we can omit it for any expression in kotlin, but It's another example when the explicit return type may be useful.
This function uses expanded control flow with null checks, so it's easy to obtain `String?` return type instead of `String` by mistake.